LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright (C) 2021-2023 Confio GmbH
```

Comment:
The LICENSE file should be unchanged. Here it includes only a template for the NOTICE file. See the section title "APPENDIX: How to apply the Apache License to your work."